behat.yml

```
@@ -101,10 +101,12 @@ legacy:
                             browserName: "chrome"
                             chrome:
                                 switches:
-                                    - "---window-size=1280,1024"
+                                    - "--window-size=1280,1024"
                                     - "--start-maximized"
                                     - "--no-sandbox"
                                     - "--headless"
+                                prefs:
+                                    foo: "Just because prefs must be a dictionnary"
```

Empty object does not work ?
```suggestion
                                prefs: {}
```

---

Let's try

---

And no, it has to be a non empty dictionary